students/gotem888/hw-12 searchEngine/css/style.css

```diff
@@ -0,0 +1,276 @@
+@property --rotate {
+    syntax: "<angle>";
+    initial-value: 132deg;
+    inherits: false;
+}
+:root {
+    --form-height: 40vh;
+    --form-width: calc(var(--form-height) / 1.5);
+    --blue: #3498db;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+    margin: 0;
+    min-height: 100vh;
+    background: #191c29;
+  }
+  
+  .wrapper {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-between;
+    align-items: flex-start;
+    gap: 30px;
+    padding: 40px 20px;
+}
+  .form {
+    background: rgba(25, 28, 41, 1);
+    display: flex;
+    flex-flow: row wrap;
+    position: relative;
+    width: 300px;
+    height: var(--card-height);
+    padding: 40px 15px;
+    border-radius: 6px;
+    font-size: 18px;
+    color: rgb(88 199 250 / 100%);
+    cursor: pointer;
```

Comment:
Consider to you `cursor: pointer;` only for interactive elements (inputs, buttons, links, etc.)